examples/main/main.cpp

```diff
@@ -153,6 +153,7 @@ int main(int argc, char ** argv) {
 
     // load the model and apply lora adapter, if any
     LOG_INF("%s: load the model and apply lora adapter, if any\n", __func__);
+    std::cout << params << "\n";
```

Comment:
This should be removed.

Reply:
Yes, sorry that's something left in from debugging I suspect. Thanks